.github/workflows/auto-assign-issue.yml

```
@@ -0,0 +1,16 @@
+name: (Privileged) Auto Assign
```

```suggestion
name: Auto Assign Issue
```
This one isn't privileged.

---

It still is (:

---

but yea, no biggie